clients/drcachesim/tracer/instru_offline.cpp

```diff
@@ -403,17 +403,23 @@ int
 offline_instru_t::append_unit_header(byte *buf_ptr, thread_id_t tid, ptr_int_t window)
```

Comment:
Does the redzone_size calculation in drmemtrace_client_main also need to be updated now? Since we have additional timestamp+cpu entries that increase the min buffer size requirement.

Reply:
I think these couple of records still fit under the 2x we do there.  I updated that comment to mention the extra timestamps.